_includes/integrations/ollama/api-endpoint.mdx

```diff
@@ -0,0 +1,3 @@
+The Weaviate server has to be able to reach the Ollama API endpoint. If Weaviate is running in a Docker container and Ollama is running on the Docker host machine, use the built-in docker `localhost` alias, `host.docker.internal`, to direct Weaviate to your local Ollama instance.
```

Comment:
Hey I think this
> Ollama is running on the Docker host machine

Should probably say:
> Ollama is running on the host machine outside of the Weaviate Docker container

Reply:
Updated